app/controllers/test/mock_socure_document_capture_controller.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module Test
+  class MockSocureDocumentCaptureController < ApplicationController
+    before_action :check_not_in_prod
+
+    def show
+      if verify?
+        simulate_socure_docv
+        return
+      end
+
+      @url = test_mock_socure_url(verify: 1, hybrid: hybrid? ? 1 : 0)
+    end
+
+    private
+
+    def check_not_in_prod
+      render_not_found if Rails.env.production?
```

Comment:
All deployed environments (including sandbox environments) are considered production. Is that the behavior we're expecting? I see we're also only registering this route if `enable_test_routes` is enabled.

I'd wonder if either (a) this isn't needed since we're only registering the route in environments with test routes enabled, or (b) we could rename this to clarify "deployed environments" instead of "production" to try to preempt any potential future misunderstandings.